src/Microsoft.Azure.ServiceBus/ServiceBusConnectionStringBuilder.cs

```diff
@@ -127,6 +154,15 @@ public string SasKey
             set => this.sasKey = value.Trim();
         }
 
+        /// <summary>
+        /// Get the transport type from the connection string
+        /// </summary>
+        public TransportType TransportType
```

Comment:
>public TransportType TransportType [](start = 8, length = 34)

Since this is changing, you will have to update the apiappovers.txt as well. Run the apiApprover test and it will generate the new file for you which you need to update.